From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] Use Digital Monster level-up learnsets

Target: rh-hideout/pokeemerald-expansion
Ref: 75b806a3ab57a81ff1eb6179288981f0b3cc3050

diff --git a/src/pokemon.c b/src/pokemon.c
--- a/src/pokemon.c
+++ b/src/pokemon.c
@@
-#if P_LVL_UP_LEARNSETS >= GEN_9
+#ifdef DIGITAL_MONSTER_SERIES
+#include "data/pokemon/level_up_learnsets/digital_monster.h"
+#elif P_LVL_UP_LEARNSETS >= GEN_9
 #include "data/pokemon/level_up_learnsets/gen_9.h" // Scarlet/Violet
@@
 #elif P_LVL_UP_LEARNSETS >= GEN_1
 #include "data/pokemon/level_up_learnsets/gen_1.h" // Yellow
 #endif
